api/utils.go

```diff
@@ -207,6 +210,116 @@ func postReplicationAction(policyID int64, acton string) error {
 	return fmt.Errorf("%d %s", resp.StatusCode, string(b))
 }
 
+// SyncRegistry syncs the repositories of registry with database.
+func SyncRegistry() error {
+
+	rc, err := initRegistryClient()
```

Comment:
could you add more logs here, such as "start syncing repositories from registry to DB"and "adding repository xxxx, deleting repositories xxxx"

Just for debug and when the bootstrap of UI is slow we can see the progress.